Pages/Index.cshtml.cs

```diff
@@ -1,47 +1,71 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.IO;
+using System.Net;
+
 
 namespace DotnetBunnyLogBrowser.Pages
 {
-	public class IndexModel : PageModel
+    public class IndexModel : PageModel
 	{
-		public string GetExampleLog() => "createdump-aspnet log:\n" + //TODO: delete example log
-			       "\n" +
-			       "+ version=2.1.504\n" +
-			       "+ dotnet new web --force\n" +
-			       "The template \"ASP.NET Core Empty\" was created successfully.\n" +
-			       "\n" +
-			       "Processing post-creation actions...\n" +
-			       "Running 'dotnet restore' on /mnt/ramdisk/dotnet-bunny/dotnet-regular-tests/createdump-aspnet/createdump-aspnet.csproj...\n" +
-			       "Restoring packages for /mnt/ramdisk/dotnet-bunny/dotnet-regular-tests/createdump-aspnet/createdump-aspnet.csproj...\n" +
-			       "Generating MSBuild file /mnt/ramdisk/dotnet-bunny/dotnet-regular-tests/createdump-aspnet/obj/createdump-aspnet.csproj.nuget.g.props.\n" +
-			       "    Generating MSBuild file /mnt/ramdisk/dotnet-bunny/dotnet-regular-tests/createdump-aspnet/obj/createdump-aspnet.csproj.nuget.g.targets.\n" +
-			       "    Restore completed in 1.22 sec for /mnt/ramdisk/dotnet-bunny/dotnet-regular-tests/createdump-aspnet/createdump-aspnet.csproj.\n" +
-			       "\n" +
-			       "Restore succeeded.\n" +
-			       "\n" +
-			       "+ dotnet tool install -g dotnet-dev-certs\n" +
-			       "Tool 'dotnet-dev-certs' is already installed.\n" +
-			       "+ true\n" +
-			       "+ dotnet dev-certs https\n" +
-			       "Cannot find command 'dotnet dev-certs', please run the following command to install\n" +
-			       "\n" +
-			       "dotnet tool install --global dotnet-dev-certs";
-
 		public List<BunnyJob> GetJobs()
 		{
-			//TODO...
+            var job_names=Directory.GetDirectories(JobsConfig.Get().JobsDirectory, JobsConfig.Get().JobsPrefix+"*");
+			var jobs=new List<BunnyJob>(job_names.Length);
+			var client=new WebClient();
 
-			return new List<BunnyJob>(){
-				new BunnyJob("dotnet-rpm-rhel7-x64-1.0", true),
-				new BunnyJob("dotnet-rpm-rhel7-x64-1.1", true),
-				new BunnyJob("dotnet-rpm-rhel7-x64-2.1", false, new List<BunnyTest>(){new BunnyTest("createdump-aspnet", GetExampleLog()), new BunnyTest("bash-completion", "Bash completion is bork and this is the example log.")}),
-				new BunnyJob("dotnet-rpm-rhel7-x64-2.2", true)
-			};
+			for(int i=0; i<job_names.Length; ++i)
+			{
+				job_names[i]=job_names[i].Substring(job_names[i].LastIndexOf('/')+1);

```

Comment:
A comment here describing why you're chopping off parts of the name here would be helpful if someone has to look at this 8 months from now with no context. Or for someone like me, who is looking at it right now with no context. :)